docfiles/pageheader.html

```diff
@@ -0,0 +1,16 @@
+<div class="ui container">
+    <div id="page-header-wrapper">
+        <div class="page-header-item page-header-content no-grow">
+            <h2 class="ui item header">@title@</h2>
+        </div>
+        <div class="page-header-item show-code-toggle">
+            <button id="show-code-button" class="ui tiny default button">Show Code</button>
+        </div>
+        <div class="page-header-item no-grow">
+            <a href="https://makecode.com/org" title="Go to Microsoft MakeCode" aria-label="Microsoft MakeCode Logo" role="menuitem" target="blank" rel="noopener" class="ui item logo organization">
+                <img class="ui logo portrait hide" src="./static/Microsoft-logo_rgb_c-gray.png" alt="Microsoft MakeCode Logo">
+                <img class="ui mini image portrait only" src="./static/Microsoft-logo_rgb_c-gray-square.png" alt="Microsoft MakeCode Logo">
```

Comment:
Same thing here, how does this affect docs. More concerned about this page. 
It's included by the package page, as well as stream.

Reply:
stream less of a concern, but package pages will be re-using this.

Reply:
Yep, I'll inline them. Good catch!